tests/testsuite.src/used_binaries.at

```diff
@@ -1102,14 +1097,13 @@ AT_CHECK([$COBC -m -fstatic-call prog.cob], [0], [], [])
 # We ignore the error output, as it depends on the C compiler in use
 
 AT_CHECK([$COBC -m --include "$PWD/file.h" -fstatic-call prog.cob], [1], [], [ignore])
+
 AT_DATA([prog2.cob], [
        IDENTIFICATION   DIVISION.
        PROGRAM-ID.      prog.
        DATA DIVISION.
-       WORKING-STORAGE SECTION.
-       01 long USAGE BINARY-C-LONG.
        PROCEDURE        DIVISION.
-           CALL "f" USING "Hello" BY VALUE long RETURNING NOTHING.
```

Comment:
So, under Windows, the function has to be available. `f` was not a real function, so it was failing with an "undefined function 'f'". I solved it by calling an existing function (`rename` is part of `stdio.h`, which is not included by default in `common.h`). My understanding was that the test was mostly testing the arity of the function, not the exact types, so I chose a function with two `char*` because I couldn't find an existing function with the same prototype as former `f`.